pkg/pillar/containerd/containerd.go

```diff
@@ -112,32 +110,23 @@ func containerdLoadImageTar(filename string) (map[string]images.Image, error) {
 	return names, nil
 }
 
-// CtrRm remove an existing container. If silent is true, then operation failures are ignored and no error is returned
-func CtrRm(containerPath string, silent bool) error {
-	log.Infof("ctrRm %s\n", containerPath)
+// SnapshotRm removes existing snapshot. If silent is true, then operation failures are ignored and no error is returned
+func SnapshotRm(containerPath string, silent bool) error {
+	log.Infof("snapshotRm %s\n", containerPath)
 
 	containerID := filepath.Base(containerPath)
-	container, err := loadContainer(containerID)
-	if err != nil {
-		err = fmt.Errorf("ctrRm: exception while loading container: %v. %v", containerID, err.Error())
+
+	if err := deleteBundle(containerID, silent); err != nil {
```

Comment:
If we don't need to create the bundle, because there is no container, then there is no need to delete it either.

As below, even if we want to do so, creating/deleting a bundle is part of the _container_ create/delete process, not the _snapshot_, so it should be outside of `SnapshotPrepare`/`SnapshotRm`

Reply:
See above -- we do more than just snapshot creation -- hence consider it an "EVE bundle" @deitch